### DISCUSSION TITLE
Markdown. Pictures

### DISCUSSION THREAD

**Discussion by SvetlanaGorelova** [requarks/wiki#4471] [Help / Questions]
Hi! 
How do you insert captions for pictures in the markdown mode? I need some captions below my images, see how it is done in the editor mode but could not find this option in the markdown mode. 
Any help will be appreciated.

**Comment by broxen** [comment]
I'd also like to know if this is possible with the current feature set.

**Comment by broxen** [comment]
Looks like this is not supported currently. There's an old PR #2191 that we can implement locally

**Comment by broxen** [comment]
There's also this markdown-it plugin: https://www.npmjs.com/package/markdown-it-figure
